src/vma/proto/route_val.cpp

```diff
@@ -45,7 +45,7 @@
 #define rt_val_logdbg		__log_info_dbg
 #define rt_val_logfunc		__log_info_func
 
-route_val::route_val(): cache_observer()
+route_val::route_val()
```

Comment:
cache_observer seem to be used previously for handling some event, however, currently this class is not handling any event.